recipe/build.sh

```diff
@@ -78,6 +78,12 @@ export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
             --with-dods-root=$PREFIX \
             $OPTS
 
+# Regenerate python bindings.
+cd swig/python
+make veryclean
+make generate
+cd ../..
+
 # CircleCI offers two cores.
 make -j 2 >> $BUILD_OUTPUT 2>&1
```

Comment:
Can you help me with another issue and change the `2` to `$CPU_COUNT` here? (`gdal` takes so long to build that I do not want to send a separate PR to fix that.)